notifications/admin.py

```diff
@@ -0,0 +1,3 @@
+from django.contrib import admin
+
+# Register your models here.
```

Comment:
I don't think we need an admin UI for the notifications. So you can just remove this bootstrap file